assets/src/components/editing/markdown_editor/markdown_util.ts

```diff
@@ -64,7 +64,7 @@ export const getMarkdownWarnings = (model: (AllModelElements | FormattedText)[])
     }
   });
 
-  const uniqueWarnings = [...new Set(warnings)];
+  const uniqueWarnings = Array.from(new Set(warnings));
```

Comment:
This, and the change over in factories.ts were the build-time errors that were not previously detected. I did attempt a build with them broken to verify those builds will now fail.